src/main/java/seedu/address/logic/commands/AddNoteCommand.java

```diff
@@ -22,17 +22,18 @@ public class AddNoteCommand extends Command {
     public static final String COMMAND_WORD = "addnote";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the note of the person whose profile matches "
-            + "the specified IC (case-insensitive). "
-            + "Existing remark will be appended by default. "
-            + "To replace the original note, add -replace at the end of your command. "
-            + "E.g. addnote i/S0123456Q n/Diabetes -replace\n"
+            + "the specified IC (case-insensitive). \n"
+            + "Existing remark will be appended by default. \n"
             + "Parameters: "
             + PREFIX_IC + "IC "
```

Comment:
maybe we can standardise with editcommand and put "Parameters: IC (must be a valid identity card number) "?

Reply:
yep! sorry overlooked that :)